src/modules/jobs.rs

```diff
@@ -1,35 +1,21 @@
 use ansi_term::Color;
-use std::process::Command;
 
 use super::{Context, Module};
 
 /// Creates a segment to show if there are any active jobs running
 pub fn segment<'a>(context: &'a Context) -> Option<Module<'a>> {
-    match get_jobs() {
-        Some(is_job) => {
-            print!("{}", is_job);
+    const JOB_CHAR: &str = "✦ ";
+    let module_color = Color::Blue.bold();
 
-            const JOB_CHAR: &str = "✦ ";
-            let module_color = Color::Blue.bold();
+    let mut module = context.new_module("jobs");
+    module.set_style(module_color);
 
-            let mut module = context.new_module("jobs");
-            module.set_style(module_color);
+    let arguments = &context.arguments;
+    if arguments.value_of("jobs").unwrap_or("0") == "0" {
+        module.new_segment("symbol", "");
```

Comment:
At this point, I'd probably suggest returning `None`.

Reply:
Sounds good @matchai!